packages/webpack/lib/configs/node.js

```diff
@@ -118,9 +118,17 @@ module.exports = function getConfig(config, name) {
       devtoolModuleFilenameTemplate: (info) =>
         resolve(info.absoluteResourcePath),
     },
-    // fixme
     cache: {
-      type: 'memory',
+      type: 'filesystem',
+      buildDependencies: {
+        config: [__filename].concat(buildDependencies),
+      },
+    },
+    snapshot: {
+      buildDependencies: {
+        hash: true,
```

Comment:
Should this not be `hash: isProduction`?

Reply:
When webpack happens to compare two equal timestamps it, it takes a look at the hashes instead. This is a nice fallback for the `node`-config, which is used in watch- in non-watch-mode I thought.